src/main/java/org/openrewrite/java/template/processor/RefasterTemplateProcessor.java

```diff
@@ -399,6 +400,27 @@ private String recipeDescriptor(JCTree.JCClassDecl classDecl, String defaultDisp
                 String displayName = defaultDisplayName;
                 String description = defaultDescription;
                 Set<String> tags = new LinkedHashSet<>();
+
+                // Extract from JavaDoc
+                Tokens.Comment comment = cu.docComments.getComment(classDecl);
+                if (comment != null && comment.getText() != null && !comment.getText().isEmpty()) {
+                    String commentText = comment.getText()
+                            .replace("{@link ", "`")
+                            .replace("}", "`")
```

Comment:
I guess `{@code` should get the same treatment.

(This wouldn't work for `{@value`, but it seems highly unlikely one would use that for a Refaster rule.)

Reply:
Thanks! Picked up in https://github.com/openrewrite/rewrite-templating/pull/51/commits/fc3bb8d87c57c47fd35306d36588d1b231880ddf to replace any such tags.